lib/tasks/dm.rake

```diff
@@ -16,6 +16,7 @@ namespace :dm do
     Rake::Task['importer:import_answers'].execute
     Rake::Task['importer:initial_featured'].execute
     Rake::Task['diffusion_history:all'].execute
+    Rake::Task['milestones:milestone_transfer'].execute
```

Comment:
I think this is suppose to be 'milestones:milestones_transfer' (the extra 's' in the second milestone)

Reply:
Updated!